src/ontology/mondo-edit.obo

```diff
@@ -537291,13 +537292,16 @@ subset: clingen {source="MONDO:CLINGEN"}
 subset: gard_rare {source="MONDO:GARD"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "RYR1-related disease" EXACT [https://rarediseases.org/rare-diseases/ryr-1-related-diseases/]
+synonym: "RYR1-related disorder" EXACT [https://rarediseases.org/rare-diseases/ryr-1-related-diseases/]
```

Comment:
@katiermullen these are exact only if ALL RYR1-related disease are myopathy. Can you please confirm that ALL mutation in RYR1 cause myopathy? Otherwise we need to make this a broad synonym.

Reply:
I found one instance where a mutation in RYR1 is related to a syndromic disease that includes but is not limited to a myopathy - 'King-Denborough syndrome'. 

I will update the exact syns to broad syns as suggested.